docs/versions/unversioned/sdk/splash-screen.md

```diff
@@ -138,7 +138,7 @@ export default class App extends React.Component {
 }
 ```
 
-## Example without any flickering between SplashScreen and it's later continueation
+## Example without any flickering between SplashScreen and it's later continuation
```

Comment:
This is only explicit change I've made, all the rest is just auto-behaviour of my IDE